docs/esa/iso/iso.rst

```diff
@@ -26,89 +24,87 @@ Examples
 1. Querying ISO data
 ------------------------------
 
-.. code-block:: python
+.. doctest-remote-data::
 
   >>> from astroquery.esa.iso import ISO
-  >>> table=ISO.query_ida_tap(query="SELECT tdt,ra,dec,aotname,target " + 
-  ...   "FROM ida.observations " + 
-  ...   "WHERE INTERSECTS(CIRCLE('ICRS',10.68470833,41.26875,0.08333333333333333)," + 
+  >>> table=ISO.query_ida_tap(query="SELECT tdt,ra,dec,aotname,target " +
+  ...   "FROM ida.observations " +
```

Comment:
instead of string additions, please add parens around the string (and use indents to line it with the start of it)